client/components/costume/EditCostume.js

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { loadSCostumeDispatch } from '../../redux/costumes/singleCostume';
-import { fetchCategories } from '../../redux/categories/allCategories';
+import { selectCategory } from '../../redux/categories/singleCategory'
 
 class EditCostume extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      id: this.props.match.params.id,
+      costId: this.props.match.params.costId,
```

Comment:
minor point but can this be called costumeId for consistency and to avoid confusion?